test/evergreen.yml

```diff
@@ -2067,6 +2067,39 @@ tasks:
             cp ../../../bench/wtperf/split_heavy.wtperf .
             ./wtperf -O ./split_heavy.wtperf -o verbose=2
 
+  - name: static-wt-build-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --disable-shared --with-builtins=snappy,zlib
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+
+            # Delete wt util
+            rm -f wt
```

Comment:
Not sure this is necessary.

Reply:
I think it is safe and prefer to delete wt util then rebuild/remake with "-static" option.

Reply:
It is relied on by the first check below. If we don't remove `wt` here, it'll still be there after a failed static compile.